Shopping_Cart/Text Files/InventoryList.txt

```diff
@@ -0,0 +1,3 @@
+﻿Banana-01-15-kg-60
```

Comment:
Put seperator as | not -
Order should be - > Code, Name, UnitsAvailable, CostPerUnit